frontend/android/gradle.properties

```diff
@@ -1,4 +1,4 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
-android.enableR8=true
+# android.enableR8=true
```

Comment:
Why not just remove it?

Reply:
Because Flutter automatically adds it again if it does not find the setting in the file (regardless of being commented out or not). 🤦🏽‍♂️
See also https://github.com/flutter/flutter/issues/52005.